pom.xml

```diff
@@ -1146,6 +1146,18 @@
                 <version>${version.junit.jupiter}</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
```

Comment:
Can we just update the pom to use the junit bom like we do in datawave-microservice-parent?  https://github.com/NationalSecurityAgency/datawave-parent/blob/main/pom.xml#L64

Reply:
I'm leaving this as an explicit dependency.

Reply:
I'll see about pulling in the `junit.bom`